koku/reporting_common/models.py

```diff
@@ -4,8 +4,42 @@
 #
 """Models for shared reporting tables."""
 from django.db import models
+from django.db.models import IntegerChoices
 from django.utils import timezone
 
+from common.enum import StrEnum
+
+
+class ReportStep(IntegerChoices):
+    DOWNLOADING = 3
+    PROCESSING = 4
+    OCP_CLOUD_PROCESSING = (5, "OCP-Cloud-Processing")
```

Comment:
Why is this one a tuple when all of the others are just integers?

Reply:
I believe its for the string conversion of `_`s the rest dont need it since they have no `_`s to convert.